SingularityUI/app/models/Request.coffee

```diff
@@ -152,7 +157,16 @@ class Request extends Model
                     return false
 
                 else
-                    localStorage.setItem(@localStorageCommandLineInputKeyPrefix + @id, commandLineInput) if commandLineInput?
+                    history = localStorage.getItem(@localStorageCommandLineInputKeyPrefix + @id)
```

Comment:
I noticed that too. It would make sense move localstorage and vex interactions into utilities. We should probably make another issue though since it would add a lot of clutter.